README.md

```diff
@@ -56,9 +61,9 @@ This opens the resource YAML file in the editor and allows the user to make the
 Pushing the changes made to the file to the cluster automatically updates the edited Kubernetes resource.
 Users can also create a new Kubernetes resource from local YAML/JSON file and push them to the cluster.
 
-> **Tip**: You're advised to edit copies of your yaml/json files. 
-> This is because the editor keeps your content in sync with the resource that exists on the cluster. 
-> Changes that happen on the cluster are included when pushing or pulling the editor.  
+> **Tip**: You're advised to edit copies of your yaml/json files.
```

Comment:
These 3 lines can be removed. This tip is not useful any more. Syncs are not done automatically but only when the user explicitly requests the server content to be pulled into the editor.